fastlite/_modidx.py

```diff
@@ -50,4 +50,5 @@
                                'fastlite.core.create_mod': ('core.html#create_mod', 'fastlite/core.py'),
```

Comment:
_Commented on notebook `fastlite/_modidx.py` Cell 1 Line 53_
```python
                               'fastlite.core._parse_typ': ('core.html#_parse_typ', 'fastlite/core.py'),
                               'fastlite.core._row': ('core.html#_row', 'fastlite/core.py'),
                               'fastlite.core._tnode': ('core.html#_tnode', 'fastlite/core.py'),
                               'fastlite.core.all_dcs': ('core.html#all_dcs', 'fastlite/core.py'),
                               'fastlite.core.create_mod': ('core.html#create_mod', 'fastlite/core.py'),
                               'fastlite.core.diagram': ('core.html#diagram', 'fastlite/core.py'),
                               'fastlite.core.get_typ': ('core.html#get_typ', 'fastlite/core.py')},
            'fastlite.db_dc': {},
```
Is there a way to exclude this, since it was just for a test?

Reply:
I'm not sure if we should leave it in or remove it.

Reply:
Don't worry about _modidx